frontend/src/pages/projects/screens/spawner/service.ts

```diff
@@ -69,7 +69,7 @@ export const replaceRootVolumesForNotebook = async (
     };
     replacedVolumeMount = { name: existingName, mountPath: ROOT_MOUNT_PATH };
   } else {
-    const pvc = await createPvc(storageData.creating, projectName, { dryRun });
+    const pvc = await createPvc(storageData.creating, projectName, undefined, { dryRun });
```

Comment:
`storageClassName` instead of undefined is ok here. See the code above that line in service.ts, I am passing storageClassName.